src/main/java/io/gatling/plugin/util/PluginReadWrite.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2011-2021 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.plugin.util;
+
+import java.util.Scanner;
+
+public interface PluginReadWrite {
+
+  public void logInfo(String message);
+
+  public void logError(String message);
+
+  public Scanner getScanner();
+}
```

Comment:
Naming seems wrong to me, proposal: 

```suggestion

public interface PluginLogger {
  public void success(String message);
  public void error(String message);
  public void error(String message, Throwable e);
}

public interface PluginScanner extends Scanner {}

public interface PluginIO {
  public PluginLogger getLogger();
  public PluginScanner getScanner();
}
```